### PR TITLE
REV-145-agent-profile-bugs

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-145

### Description

- fix agent details page bugs

### CODE REVIEW

File: src/components/agent-details/AgentInformation/index.jsx [hunk 1]
```diff
@@ -46,6 +51,7 @@ const companyInfoIcons = [<CompanyName />, <EmailIcon />, <CompanyWebsite />];
 function AgentInformation() {
   const { id } = useParams();
   const [userData, setUserData] = useState({});
+  const [likeActive, setLikeActive] = useState(false);
```
Comment (mohdabed123) [inline src/components/agent-details/AgentInformation/index.jsx:54]:
```suggestion
  const [isLiked, setIsLiked] = useState(false);
```

File: src/components/agent-details/AgentRating/index.jsx [hunk 2]
```diff
@@ -52,8 +53,15 @@ function AgentRating() {
...
-              {value ? <Like /> : <EmptyLike />}
-              {name}
+              <span>{name}</span>
+              {value ? (
+                <Like />
+              ) : (
+                <IconWrapper>
+                  <Like />
+                  <EmptyLike />
+                </IconWrapper>
```
Comment (mohdabed123) [inline src/components/agent-details/AgentRating/index.jsx:63]: i see that this is getting redundant, can we make we make to be a shared component.

File: src/components/agent-details/CardInformation/index.jsx [hunk 3]
```diff
@@ -47,7 +47,10 @@ function CardInformation({
             ${price?.toLocaleString()}
             {perPeriod && `/${perPeriod}`}
           </h3>
-          <span>{getDatesDifference(createdAt)} days on reverifi</span>
+          <span>
+            {getDatesDifference(createdAt)} day
+            <span>{getDatesDifference(createdAt) > 1 && 's'}</span> on reverifi
```
Comment (mohdabed123) [inline src/components/agent-details/CardInformation/index.jsx:52]:
```suggestion
            {getDatesDifference(createdAt) > 1 && 's'} on reverifi
```
is there a reason for using span just for the 's'?

File: src/components/agent-details/AgentListings/agent-listings.style.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ export const CardsContainer = styled.div`
   gap: 1rem;
 
   > div {
-    flex: 1;
+    flex: ${({ cardsNum }) => cardsNum > 1 && 1};
```
Comment (mohdabed123) [inline src/components/agent-details/AgentListings/agent-listings.style.js:19]:
```suggestion
   ${({ cardsNum }) => cardsNum >1 && `
    flex: 1;
  `}
```

**Review by mohdabed123 — CHANGES_REQUESTED** [review]
Looks good just small changes

File: src/components/agent-details/AgentInformation/agent-information.style.js [hunk 5]
```diff
@@ -71,8 +75,9 @@ export const ContactInfo = styled.div`
   gap: 1.4rem;
 `;
 
-export const InfoWrapper = styled.li`
+export const InfoWrapper = styled.div`
   display: flex;
+  gap: ${({ company }) => (company ? 0 : '3rem')};
```
Comment (AhmadAlShwaiki) [inline src/components/agent-details/AgentInformation/agent-information.style.js:80]: confused by this prop name, please change it if possible.

File: src/components/agent-details/AgentInformation/index.jsx [hunk 6]
```diff
@@ -15,7 +14,12 @@ import useEffectOnce from 'hooks/use-effect-once';
 import React, { useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { getUserInfo } from 'services/user';
-import { separateBy, toUpperCaseFirstLetter } from 'utils/helpers';
+import {
+  formatPhoneNumber,
+  separateBy,
+  toUpperCaseFirstLetter,
+} from 'utils/helpers';
+import LikeButton from '../../shared/LikeButton';
```
Comment (AhmadAlShwaiki) [inline src/components/agent-details/AgentInformation/index.jsx:22]:
```suggestion
import LikeButton from 'components/shared/LikeButton';
```

File: src/components/agent-details/AgentInformation/agent-information.style.js [hunk 7]
```diff
@@ -86,16 +91,23 @@ export const InfoWrapper = styled.li`
 export const InfoKey = styled.div`
   align-items: center;
   display: flex;
-  flex: 2;
+  min-width: 9rem;
+  /* flex: 1; */
```
Comment (AhmadAlShwaiki) [inline src/components/agent-details/AgentInformation/agent-information.style.js:95]:
```suggestion
```

File: src/components/agent-details/CardInformation/index.jsx [hunk 8]
```diff
@@ -77,16 +80,16 @@ function CardInformation({
...
           )}
-          {homeArea && (
+          {homeArea ? (
             <IconGroup>
               <span>{homeArea} sqft</span>
             </IconGroup>
-          )}
-          {lotArea && (
+          ) : null}
+          {lotArea ? (
```
Comment (AhmadAlShwaiki) [inline src/components/agent-details/CardInformation/index.jsx:88]: Does this make any difference?

**Review by mohdabed123 — APPROVED** [review]

File: src/components/agent-details/CardInformation/index.jsx (hunk 8 above)
Comment (IbrahimAldereni-socuimtech) [reply src/components/agent-details/CardInformation/index.jsx:88]: yes it will show (0) on the frontend if it's falsy

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**